.editorconfig

```diff
@@ -2,3 +2,14 @@
 
 # ProhibitedMemberRule: Prohibited Type Or Member
 dotnet_diagnostic.ProhibitedMemberRule.severity = error
+csharp_new_line_before_open_brace=all
+csharp_new_line_before_members_in_object_initializers=false
+csharp_new_line_before_members_in_anonymous_types=false
+csharp_new_line_between_query_expression_clauses=false
+insert_final_newline=true
+csharp_new_line_before_else=true
+
+[*.{cs,vb}]
+indent_size=4
+tab_width=4
+end_of_line=lf
```

Comment:
Can you remove these from the CR?

Reply:
these damn files creep in all the time ...